core/embed/projects/prodtest/cmd/prodtest_secrets.c

```diff
@@ -128,6 +129,73 @@ static void prodtest_secrets_init(cli_t* cli) {
   cli_ok(cli, "");
 }
 
+#ifdef SECRET_MASTER_KEY_SLOT_SIZE
+static void prodtest_secrets_get_mcu_device_key(cli_t* cli) {
+  if (cli_arg_count(cli) > 0) {
+    cli_error_arg_count(cli);
+    return;
+  }
+
+  ed25519_secret_key mcu_private = {0};
```

Comment:
Good catch with the incorrect type. Could you please also fix the type in `secret_key_mcu_device_auth(curve25519_key dest)` in secret_keys.h, stm32u5/secret_keys.c and unix/secret_keys.c?